texar/data/data/multi_aligned_data.py

```diff
@@ -267,102 +277,101 @@ def _get_filter(delimiter, max_seq_length):
-
-                        If specified, the :attr:`"embedding_init"` field of the
-                        current dataset is ignored. Default is `None` which
-                        disables the initial value sharing.
-
-                    `"processing_share_with"`: int, optional
-                        Share the processing configurations of a preceding text
-                        dataset with the specified index in the list (starting
-                        from 0). The specified dataset must be a text dataset,
-                        and must have an index smaller than the current dataset.
-
-                        If specified, relevant field of the current dataset are
-                        ignored, including `delimiter`, `bos_token`,
-                        `eos_token`, and "other_transformations". Default is
-                        `None` which disables the processing sharing.
-
-            2. For the **general** hyperparameters, see
-            :meth:`texar.data.DataBase.default_hparams` for details.
+        .. code-block:: python
+
+            {
+                # (1) Hyperparams specific to text dataset
+                "datasets": []
+                # (2) General hyperparams
+                "num_epochs": 1,
+                "batch_size": 64,
+                "allow_smaller_final_batch": True,
+                "shuffle": True,
+                "shuffle_buffer_size": None,
+                "shard_and_shuffle": False,
+                "num_parallel_calls": 1,
+                "prefetch_buffer_size": 0,
+                "max_dataset_size": -1,
+                "seed": None,
+                "name": "multi_aligned_data",
+            }
+
+        Here:
+
+        1. "datasets" is a list of `dict` each of which specifies a
+        dataset which can be text, scalar or Record. The :attr:`"data_name"`
+        field of each dataset is used as the name prefix of the data fields
+        from the respective dataset. The :attr:`"data_name"` field of each
+        dataset should not be the same.
+
+            i) For scalar dataset, the allowed hyperparameters and default
+            values are the same as the "dataset" field of
+            :meth:`texar.data.ScalarData.default_hparams`. Note that
+            :attr:`"data_type"` must be explicitly specified
+            (either "int" or "float").
+
+            ii) For Record dataset, the allowed hyperparameters and default
+            values are the same as the "dataset" field of
+            :meth:`texar.data.RecordData.default_hparams`. Note that
+            :attr:`"data_type"` must be explicitly specified ("record").
+
+            iii) For text dataset, the allowed hyperparameters and default
+            values are the same as the "dataset" filed of
+            :meth:`texar.data.MonoTextData.default_hparams`,
+            with several extra hyperparameters:
+
+                `"data_type"`: str
```

Comment:
And these should be indented to the same column as the main body of text in this list item.